src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class SarifErrorListItemTests
+    {
+        [Fact]
+        public void SarifErrorListItem_LineMarker_Region_StartLine()
+        {
+            new SarifViewerPackage();
```

Comment:
In that case:
1. Introduce a parameterless constructor:
```
public SarifErrorListItem()
{
    new SarifViewerPackage();
}
```
xUnit will create a new instance to run each test, so effectively this becomes "common test startup code".
2. Add a comment to the ctor explaining why this is needed.

---
In reply to: [135291088](https://github.com/Microsoft/sarif-sdk/pull/595#discussion_r135291088) [](ancestors = 135291088)

Reply:
The SarifErrorListItem class already has a parameterless constructor. My issue is that I don't want to declare a "new SarifViewerPackage()" in the actual project code, since I don't want to overwrite anything that is done elsewhere. I only need it to initialize one field for these tests (and that field is not even used during the test, its just read and set into a member of another class). Not sure how we should proceed.

---
In reply to: [135293512](https://github.com/Microsoft/sarif-sdk/pull/595#discussion_r135293512) [](ancestors = 135293512,135291088)

Reply:
My mistake. I meant to add a parameterless ctor for `SarifErrorListItemTests`!

---
In reply to: [135297993](https://github.com/Microsoft/sarif-sdk/pull/595#discussion_r135297993) [](ancestors = 135297993,135293512,135291088)